### PR TITLE
Update organizer page with 2024 team

### PR DESCRIPTION
**Pull request by Rubiginosa**

### CODE REVIEW

File: organizers/index.html [hunk 1]
```diff
@@ -4,60 +4,40 @@
...
-    pos: Vice President of Outreach
-    pronouns: He/Him
+    pos: President
   - 
-    name: Roman Raguso
-    email: rraguso1@binghamton.edu
-    pos: Vice President of Communications
-    pronouns: He/Him
+    name: Lucy Loerker
+    email: lucy@hackbu.org
```
Comment (CodingKoopa) [inline organizers/index.html]: this email no longer exists (though I think it does redirect to `admin@hackbu.org`) - you can use `kloerke1@binghamton.edu`

**Review by CodingKoopa — CHANGES_REQUESTED** [review]
- Have you been able to test this locally?
- Also can now be updated for the new organizers :)

File: organizers/index.html [hunk 2]
```diff
@@ -4,60 +4,40 @@
...
-    pos: President
-    pronouns: She/Her
-  - 
-    name: Jonathan Shin
-    email: jshin75@binghamton.edu
-    pos: Vice President of Logistics
-    pronouns: He/Him
   - 
     name: Christopher Bingham
     email: cbingha@binghamton.edu
```
Comment (CodingKoopa) [inline organizers/index.html]: perhaps `admin@hackbu.org`

File: organizers/index.html [hunk 3]
```diff
@@ -5,59 +5,57 @@
...
-    email: lucy@hackbu.org
+    name: Christopher Bingham
+    email: admin@hackbu.org
     pos: President
-    pronouns: She/Her
-  - 
-    name: Jonathan Shin
-    email: jshin75@binghamton.edu
-    pos: Vice President of Logistics
-    pronouns: He/Him
```
Comment (CodingKoopa) [inline organizers/index.html:16]: If you're getting rid of the pronouns, you should also remove the relevant CSS: https://github.com/hackbinghamton/hackbinghamton.github.io/blob/d225137b52b3ba77d8c9ce45d16d29d498175a98/less/main.less#L529-L544

**Review by CodingKoopa — CHANGES_REQUESTED** [review]

File: organizers/index.html (hunk 3 above)
Comment (Rubiginosa) [reply organizers/index.html:16]: Collecting them's been kind of a pain in the past—I'll remove it for now and we can add them back once we're more on track.

**Pull request merged**